src/core/thread/context.d

```diff
@@ -72,6 +138,41 @@ class StackContextExecutor
             break;
         }
     }
+    
```

Comment:
```
src/core/thread/context.d:141:    
posix.mak:361: recipe for target 'checkwhitespace' failed
```
???

Reply:
No worries, I usually forget to check style. :)

Reply:
The `???` was not aimed at you, sorry. Usually it complains about spaces before the end of a line, bt as far as i ca tell there is no whitespace there at all! Oh well, I'm sure you'll figure out how to appease that useless checker somehow. 'Night.

Reply:
Hahaha. I probably have fought the style checker as few, but here, there is space. :P 
You know, it's: ---\n (where - is a whitespace) instead of: \n
Style checker - Stefanos: 1-0
Good night.